pkg/operators/v1alpha1/clusterserviceversion_types.go

```diff
@@ -148,8 +148,11 @@ type APIServiceDescription struct {
 // APIResourceReference is a Kubernetes resource type used by a custom resource
 // +k8s:openapi-gen=true
 type APIResourceReference struct {
-	Name    string `json:"name"`
-	Kind    string `json:"kind"`
+	// Name of the CustomResourceDefinition for the referent, or an empty string when the referent is not a custom resource.
```

Comment:
Is this `metadata.name` of the CRD? If possible, it might be nice to be a little more explicit about where these are supposed to come from if its a simple mapping from CRD fields.

Reply:
Yeah, this field is the confusing one. I can update to 'metadata.name' instead of 'Name'